core/modules/node/templates/node.tpl.php

```diff
@@ -99,7 +99,7 @@
     </footer>
   <?php endif; ?>
 
-  <div class="content"<?php print backdrop_attributes($content_attributes); ?>>
+  <div class="content clearfix"<?php print backdrop_attributes($content_attributes); ?>>
```

Comment:
...wondering if there should be a space between the last double quote, after the classes declaration, and before the printing of `$content_attributes`?

Reply:
...and also whether the `content` and `clearfix` classes should be merged into any possible classes provided via `$content_attributes`.